pages/conduct/changes.md

```diff
@@ -0,0 +1,22 @@
+<script>{
+  "title": "jQuery Foundation Code of Conduct - Changes",
+  "pageTemplate": "page-conduct.php"
+}</script>
+
+## Change control process
+
+We're (mostly) programmers, so we'll track changes to the code of conduct and associated documents the same way we track changes to code. All changes will proposed via a pull request to the [jquery.org repository on GitHub](http://github.com/jquery/jquery.org). Changes will be reviewed by the conduct committee first, and then sent to the jQuery Foundation, the jQuery development team, and the jQuery community for comment. We'll hold a comment period of at least one week, and then each group will vote on the change using its normal process (a board for the jQuery Foundation, [a core dev vote](https://docs.jquery.org/en/dev/internals/contributing/bugs-and-features/#how-we-make-decisions) for the core team). Approved changes will be merged, published, and noted below.
+
+This only applies to material changes; changes that don't effect the intent (typo fixes, re-wordings, etc.) can be made immediately.
+
+A complete list of changes can always be found [on GitHub](https://github.com/jquery/jquery.org/commits/master/pages/conduct.md); major changes and releases are summarized below.
+
+## Changelog
```

Comment:
Actually looks like the link is added up above, so I'm just going to remove the changelog section if that works with everyone